server/api/schema.go

```diff
@@ -2,11 +2,21 @@ package api
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"strconv"
 
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/odpf/stencil/server/domain"
 	stencilv1beta1 "github.com/odpf/stencil/server/odpf/stencil/v1beta1"
 )
 
+var (
```

Comment:
Didn't find any uses for this.  If we are not using then let's remove

Reply:
yes. We can remove this. made the change accordingly